### ISSUE TITLE
Error in banner image?

### ISSUE THREAD

**Issue by brentenebros** [rLoopTeam/website-v5#37]
On far right of banner image, there appears to be a vertical line. Not sure what is causing this or if it can be fixed.
![capture2](https://cloud.githubusercontent.com/assets/13357722/25665708/614693b0-2fec-11e7-9aad-176d46ffa545.JPG)

**Comment by AsLogd** [comment]
Yes this is in the image. @aksharagarwal said that he would try to fix it with photoshop

**Comment by AAgar** [comment]
I wasn't able to remove the line without blurring certain parts of the image around the line. I will just try replacing the background image
